src/gateway_gatt_char_assert_loc.erl

```diff
@@ -48,39 +47,19 @@ write_value(State = #state{}, Bin) ->
         #gateway_assert_loc_v1_pb{
             lat = Lat,
             lon = Lon,
-            owner = Owner,
-            nonce = Nonce,
-            fee = Fee,
-            amount = Amount,
-            payer = Payer
+            owner = _Owner,
+            nonce = _Nonce,
+            fee = _Fee,
+            amount = _Amount,
+            payer = _Payer
         } ->
             H3Index = h3:from_geo({Lat, Lon}, ?H3_LATLON_RESOLUTION),
             H3String = h3:to_string(H3Index),
             lager:info(
-                "Requesting assert_loc_txn for lat/lon/acc: {~p, ~p, ~p} index: ~p",
+                "Requesting unsupported assert_loc_txn for lat/lon/acc: {~p, ~p, ~p} index: ~p",
                 [Lat, Lon, ?H3_LATLON_RESOLUTION, H3String]
             ),
-            Value =
-                case
-                    ebus_proxy:call(
-                        State#state.proxy,
-                        "/",
-                        ?MINER_OBJECT(?MINER_MEMBER_ASSERT_LOC),
-                        [string, string, uint64, uint64, uint64, string],
-                        [H3String, Owner, Nonce, Amount, Fee, Payer]
-                    )
-                of
-                    {ok, [BinTxn]} ->
-                        BinTxn;
-                    {error, Error} ->
-                        lager:warning("Failed to get assert_loc txn: ~p", [Error]),
-                        case Error of
-                            "org.freedesktop.DBus.Error.ServiceUnknown" -> <<"wait">>;
-                            ?MINER_ERROR_BADARGS -> <<"badargs">>;
-                            ?MINER_ERROR_INTERNAL -> <<"error">>;
-                            _ -> <<"unknown">>
-                        end
-                end,
+            Value = <<"error">>,
```

Comment:
I had `<<"unsupported">>` there originally, but it would likely break the apps. It's not longer being used by any recent app so this is just to error out the legacy versions with something they can handle